server/db/create_test_data.js

```diff
@@ -4,14 +4,19 @@
 
 // this script assumes you've run the create_db script.
 
+//jscs:disable
```

Comment:
lol, I couldn't figure out how to get jscs to ignore the double quotes. I'll fix.